README.md

```diff
@@ -27,12 +50,198 @@ $ composer require league/uri-parser
 Documentation
 ---------
 
-Full documentation can be found at [uri.thephpleague.com](http://uri.thephpleague.com).
+### URI Parsing
+
+~~~php
```

Comment:
Is this a typo (to not use backtick)?

Reply:
not a typo both are supported by Jenkyll but I'll normalized